examples/cactus-example-supply-chain-business-logic-plugin/src/main/typescript/business-logic-plugin/supply-chain-cactus-plugin.ts

```diff
@@ -71,7 +91,8 @@ export class SupplyChainCactusPlugin
       contractAddress: this.options.contracts.bambooHarvestRepository.address,
       contractAbi: this.options.contracts.bambooHarvestRepository.abi,
       apiClient: this.options.quorumApiClient,
-      web3SigningCredential: this.options.web3SigningCredential,
+      web3SigningCredential: this.options
+        .web3SigningCredential as Web3SigningCredential,
```

Comment:
@jordigiam I'm guessing the casting here was necessary because the options object has the `web3SigningCredential` property declared as optional. Could it be that it's more fitting to change that from optional to mandatory and then no cast is necessary (plus the caller who passes in the options object is informed that way that we do assume the credentials to be present)

Reply:
If I change that from optional to mandatory the Fabric busines logic with the InsertShipmentEndpoint doesn't work, because the Fabric Ledger doesn't have the Web3SigningCredential. For now, I will keep the casting.

Reply:
Oh, my bad, forgot that we use the same plugin for all of the ledgers, apologies!